sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -38,7 +38,7 @@
         public static final String USER_AGENT = "User-Agent";
         public static final String IF_MODIFIED_SINCE = "If-Modified-Since";
         public static final String IF_MATCH = "If-Match";
-        public static final String IF_NONE_MATCH = "If-NONE-Match";
+        public static final String IF_NONE_MATCH = "If-None-Match";
```

Comment:
Does casing matters or just as convention?

Reply:
Http headers should be case-insensitive - although Gateway had a couple of issues with it I believe. Anyway - the motivation for this change was to be consistent with Backend and .Net.